public/svg/RightArrow.tsx

```diff
@@ -0,0 +1,18 @@
+export function RightArrow() {
+  return (
+    <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="12"
+    height="12"
+    viewBox="0 0 12 12"
+    fill="none"
+  >
+    <path
+      d="M7.45323 6L4.16412 9.28913C3.97863 9.47462 3.97863 9.77538 4.16412 9.96088C4.34962 10.1464 4.65038 10.1464 4.83587 9.96088L4.76561 9.89061L4.83587 9.96088L8.46086 6.33586C8.64636 6.15036 8.64636 5.84964 8.46086 5.66414L4.83588 2.03914C4.65038 1.85364 4.34962 1.85364 4.16412 2.03914C3.97863 2.22464 3.97863 2.52536 4.16412 2.71086L4.23483 2.64015L4.16412 2.71086L7.45323 6Z"
+      fill="#8C8C8C"
+      stroke="#8C8C8C"
+      stroke-width="0.2"
```

Comment:
```suggestion
      strokeWidth="0.2"
```
해당 부분도 카멜 케이스로 변경해 주세요!